packages/dds/tree/src/codec/codec.ts

```diff
@@ -344,17 +360,47 @@ export function withSchemaValidation<
  * @alpha
  */
 export enum FluidClientVersion {
+	/**
+	 * Fluid Framework Client 1.4 and newer.
+	 * @remarks
+	 * This opts into support for the 1.4 LTS branch.
+	 * @privateRemarks
+	 * As long as this code is in Tree, there is no reason to have this option as SharedTree did not exist in 1.4.
+	 */
+	// v1_4 = 1.4,
+
 	/** Fluid Framework Client 2.0 and newer. */
-	v2_0 = "v2_0",
+	v2_0 = 2.0,
+
 	/** Fluid Framework Client 2.1 and newer. */
-	v2_1 = "v2_1",
-	/** Fluid Framework Client 2.2 and newer. */
-	v2_2 = "v2_2",
-	/** Fluid Framework Client 2.3 and newer. */
-	v2_3 = "v2_3",
+	// If we thing we might want to start allowing opting into something that landed in 2.1 (without opting into something newer),
+	// we could add an entry like this to allow users to indicate that they can be opted in once we are ready,
+	// then update it to "v2_1" once we actually have the opt in working.
+	// v2_1 = v2_0,
+
+	/** Fluid Framework Client 2.41 and newer. */
+	// If we land some new formats in 2.41, we can enable selecting
+	// v2_41 = 2.41,
+
+	/**
+	 * Enable unreleased and unfinished features.
+	 * @remarks
+	 * Using this value can result in documents which can not be opened in future versions of the framework.
+	 * It can also result in data corruption by enabling unfinished features which may not handle all cases correctly.
+	 *
+	 * This can be used with specific APIs when the caller has knowledge of what specific features those APIs will be oped into with it.
```

Comment:
```suggestion
	 * This can be used with specific APIs when the caller has knowledge of what specific features those APIs will be opted into with it.
```